server/gql/models/vehicles/index.js

```diff
@@ -121,3 +122,16 @@ export const vehicleMutations = {
   type: GraphQLVehicle,
   model: db.vehicleModel
 };
+
+function _formatForOrderByClause(args) {
```

Comment:
This should be reusable and unit tested.

Reply:
I am adding this in the other file and will cover the Unit test in next PR.